release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -135,6 +135,11 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
 [id="ocp-4-10-web-console"]
 === Web console
 
+[id="ocp-cluster-support-level"]
+==== Cluster support level
+
+{product-title} now allows you to easily view support level information about your cluster via an alert on the *Overview* -> *Cluster* page, in the *Cluster Settings*, in the *About* modal, and adds a notification to your notifications drawer when the cluster is unsupported. You can manage subscription settings by clicking *Manage Subscription Settings* under the *Service Level Agreement (SLA)* of the *Overview* page.
```

Comment:
I'm afraid we don't have `Overview -> Cluster page`, are you targetting `Overview -> Details card` or something else?

Reply:
@yapei yes, I think so. I have updated and moved to the known issues section of the release notes.